### PR TITLE
Change instructions on DDH

### PR DESCRIPTION
**Pull request by jagtalon**
For @MariagraziaAlastra 

screen shot 2016-11-16 at 4 26 24 pm

### CODE REVIEW

File: share/site/duckduckhack/index_content.tx [hunk 1]
```diff
@@ -24,81 +24,52 @@
...
-                    <span class="eighty t-xxl ddh--help__text"><: l("How do I get started with DuckPan? What is DuckPan?") :></span>
-                </a>
-            </div>
+  <div class="cw--c">
+    <div class="">
+      <div class="ddh--help__section">
+        <i class="ddgsi ddgsi-check-sign"></i>
+        <h2 class="ddh__title">Get Started</h2>
+        <p>
+          Improve Instant Answers or create new ones.
```
Comment (MariagraziaAlastra) [inline share/site/duckduckhack/index_content.tx]: @jagtalon Could you remove periods from paragraphs as well?

**Comment by russellholt** [comment]
How about:

---

**Get Started**

The DuckDuckHack community is focused on improving search results for the top 15 programming languages. Our goal is to have a high quality, relevant Instant Answer for every search.

-> [Choose a task](..)
Improve existing Instant Answers or Create New ones

---

then:

- move Join Slack to the bottom of the Discuss section.
- remove "help define..."


@moollaza @jdorweiler

**Comment by moollaza** [comment]
- "Instant Answers by Language" is a bit vague. The phrasing we have on the current Getting Started is more descriptive: "Current projects and Instant Answers in each language"

- Similarly, "Become a Language Leader" needs more context. It should be clarified that it's for language experts.

**Comment by russellholt** [comment]
@moollaza 

Okay. If it's not too long, we can expand that link. It could be

- [Current Projects and Instant Answers](link) in each langauge

**Comment by russellholt** [comment]
As for the text of Become a Language Leader, we could copy the text from [Getting Started](https://forum.duckduckhack.com/t/duckduckhack-getting-started/53)

If you are an experienced DuckDuckHack contributor,
or are an expert in one of the languages we're working on, see:

- [Become a Language Leader](link)

**Comment by jagtalon** [comment]
@russellholt @moollaza @MariagraziaAlastra @jdorweiler 

What do you think?

screen shot 2016-11-16 at 5 49 18 pm

**Comment by MariagraziaAlastra** [comment]
LGTM 👍

File: share/site/duckduckgo/base.tx [hunk 2]
```diff
@@ -9,7 +9,7 @@
   <head>
     <: include "head.tx" :>
   </head>
-  <body id="pg-<: $f.filebase :>" class="page-<: $f.filebase :><: if $homepage { :> body--home<: } :>">
```
Comment (bsstoner) [inline share/site/duckduckgo/base.tx]: @jagtalon @MariagraziaAlastra Why did the `beta` class get added to the homepage in production?!

Comment (jagtalon) [inline share/site/duckduckgo/base.tx]: Oh wow I think what happened was that I pushed into an existing branch. Not sure if that branch existed on my machine or remotely. 

Will pick off those commits.

Comment (jagtalon) [inline share/site/duckduckgo/base.tx]: @bsstoner @MariagraziaAlastra https://github.com/duckduckgo/duckduckgo-publisher/pull/300

**Pull request merged**